pkgs/development/tools/dapper/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoPackage
+, lib
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildGoPackage rec {
+  name = "dapper-${version}";
```

Comment:
Can be changed to: `pname = "dapper";`, name will be built from pname and version

Reply:
updated